frontend/common/src/components/SideMenu/SideMenu.tsx

```diff
@@ -45,35 +45,27 @@ const SideMenu: React.FC<SideMenuProps> = ({
   };
 
   return !isSmallScreen || isOpen ? (
-    <FocusLock
-      autoFocus
-      returnFocus
-      disabled={!isSmallScreen}
-      className={`side-menu${isOpen ? ` side-menu--open` : ``}`}
+    <div
+      data-h2-flex-item="base(content)"
+      data-h2-background-color="base(dt-secondary.light)"
     >
-      <RemoveScroll
-        enabled={isSmallScreen && isOpen}
-        data-h2-padding="b(all, xs)"
-        data-h2-overflow="b(y, auto)"
-        data-h2-bg-color="b(lightnavy)"
-        data-h2-shadow="m(m)"
-        data-h2-position="b(fixed)"
-        data-h2-width="b(100)"
+      <FocusLock
+        autoFocus
```

Comment:
Changes here seem to cause a loss of some scroll behaviour that i think @esizer put some time into (sticky top "close" and bottom "login/logout" and scrollable for middle part of menu)

Reply:
So I actually changed this deliberately - I understand what the intent was with the original scroll effect, but I was concerned that it was preventing the affordance of scrolling by pinning an item to the bottom of the viewport - this essentially made it look like the menu couldn't be scrolled at all in some cases.

Reply:
I'm confident this is related to the issue @mnigh found with the weird scrolling issue on skill families page. So assuming the scroll issue was dealt with, that issue should also be resolved.

https://github.com/GCTC-NTGC/gc-digital-talent/pull/3051#issuecomment-1219573474